Gèrer l'erreur manque de face data.

gaggaetan opened the pull request.

gaggaetan committed:
Gèrer l'erreur manque de face data.

gaggaetan committed:
Check les face_data une fois les users récup

IchikyOtsu reviewed (approved):
La reconnaissance Fonctionne mais quand une UD manque, il la régénère bien mais ne reconnais plus la personne.

IchikyOtsu committed:
Changement de la méthode de vérification des présences

Placement en dehors de la boucle principale. Bug fix de la face data locale qui se met pas à jour.
